<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>678</width>
         <height>764</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progress_OK">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead_ChemCam_Data"/>
    <addaction name="actionLoad_Data"/>
    <addaction name="actionSet_Output_Path"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Current_Data"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_New_Workflow"/>
    <addaction name="actionRestore_Workflow"/>
    <addaction name="actionSave_Current_Workflow"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPreprocessing">
    <property name="title">
     <string>Preprocessing</string>
    </property>
    <addaction name="actionRemove_Rows"/>
    <addaction name="actionSplit_Data"/>
    <addaction name="actionInterpolate"/>
    <addaction name="actionRemove_Baseline"/>
    <addaction name="actionApply_Mask"/>
    <addaction name="actionPeak_Areas"/>
    <addaction name="actionMultiply_by_Vector"/>
    <addaction name="actionNormalization"/>
    <addaction name="actionDimensionality_Reduction"/>
    <addaction name="actionStratified_Folds"/>
   </widget>
   <widget class="QMenu" name="menuRegression">
    <property name="title">
     <string>Regression</string>
    </property>
    <addaction name="actionCross_Validation"/>
    <addaction name="actionTrain"/>
    <addaction name="actionSubmodel_Predict"/>
    <addaction name="actionPredict"/>
   </widget>
   <widget class="QMenu" name="menuVisualization">
    <property name="title">
     <string>Visualization</string>
    </property>
    <addaction name="actionPlot"/>
    <addaction name="actionPlot_Spectra"/>
    <addaction name="actionPlot_ICA_PCA"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreprocessing"/>
   <addaction name="menuRegression"/>
   <addaction name="menuVisualization"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_ChemCam_Data">
   <property name="text">
    <string>Read ChemCam Data</string>
   </property>
  </action>
  <action name="actionLoad_Data">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionSet_Output_Path">
   <property name="text">
    <string>Set Output Path</string>
   </property>
  </action>
  <action name="actionSave_Current_Data">
   <property name="text">
    <string>Save Current Data</string>
   </property>
  </action>
  <action name="actionCreate_New_Workflow">
   <property name="text">
    <string>Create New Workflow</string>
   </property>
  </action>
  <action name="actionRestore_Workflow">
   <property name="text">
    <string>Restore Workflow</string>
   </property>
  </action>
  <action name="actionSave_Current_Workflow">
   <property name="text">
    <string>Save Current Workflow</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove_Rows">
   <property name="text">
    <string>Remove Rows</string>
   </property>
  </action>
  <action name="actionSplit_Data">
   <property name="text">
    <string>Split Data</string>
   </property>
  </action>
  <action name="actionInterpolate">
   <property name="text">
    <string>Interpolate</string>
   </property>
  </action>
  <action name="actionRemove_Baseline">
   <property name="text">
    <string>Remove Baseline</string>
   </property>
  </action>
  <action name="actionApply_Mask">
   <property name="text">
    <string>Apply Mask</string>
   </property>
  </action>
  <action name="actionPeak_Areas">
   <property name="text">
    <string>Peak Areas</string>
   </property>
  </action>
  <action name="actionMultiply_by_Vector">
   <property name="text">
    <string>Multiply by Vector</string>
   </property>
  </action>
  <action name="actionNormalization">
   <property name="text">
    <string>Normalization</string>
   </property>
  </action>
  <action name="actionDimensionality_Reduction">
   <property name="text">
    <string>Dimensionality Reduction</string>
   </property>
  </action>
  <action name="actionStratified_Folds">
   <property name="text">
    <string>Stratified Folds</string>
   </property>
  </action>
  <action name="actionCross_Validation">
   <property name="text">
    <string>Cross Validation</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="actionSubmodel_Predict">
   <property name="text">
    <string>Submodel Predict</string>
   </property>
  </action>
  <action name="actionPredict">
   <property name="text">
    <string>Predict</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionPlot_Spectra">
   <property name="text">
    <string>Plot Spectra</string>
   </property>
  </action>
  <action name="actionPlot_ICA_PCA">
   <property name="text">
    <string>Plot ICA/PCA</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
